MetaCell/salk-interactive-atlas
# Feature/fixes

Applies a couple of small fixes that didn't get an issue:
- Fixes previous widget error (reported by @zsinnema )
- Changes webpack to allow geppetto source maps when running geppetto locally (thanks @ddelpiano )
- Disables dragging to prevent empty tabset left on dragging (to be properly fix in geppetto in the future)
- Cleans up explorer viewer to allow focus of the customer on the actual features available (requested by @ddelpiano )

How to test this PR: 
- You should **not** see this error in the console when opening an experiment:
![image](https://user-images.githubusercontent.com/19196034/159978556-ef2df11f-2caa-4861-891a-3a4a36625cce.png)

- You should not be able to drag widgets

- You should only see My Experiments in the explorer viewer

### Sanity checks:

### Breaking changes (select one):

### Possible deployment updates issues (select one):

### Test coverage (select one):

### Documentation (select one):
